backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/appsettings.AT21.json

```diff
@@ -10,5 +10,18 @@
   "GeneralSettings": {
     "FrontendBaseUrl": "https://am.ui.at21.altinn.cloud",
     "Hostname": "at21.altinn.cloud"
+  },
+  "KeyVaultSettings": {
+    "SecretUri": "https://altinn-at21-am-kv.vault.azure.net/"
+  },
+  "ClientSettings": {
```

Comment:
For ClientSettings, 3/4 values are common and could be setup in the non-env appsettings.json and the fourth is the same as KeyVaultSettings URI

Reply:
Moved them to appsettings